### ISSUE TITLE
[Docs]: services.md refers to invalid signature (array instead of {in: [...]} ) for exclusion and inclusion validators

### ISSUE THREAD

**Issue by taivo** [redwoodjs/redwood#7481]
### Summary and description

Documentation for service validation refers to a short form of `exclusion` and `inclusion` where only an array needs to be specified, like this:

```
validate(input.name, 'Name', {
  exclusion: ['Admin', 'Owner']
})
```

However, it does not look like this form is supported. I can see in VSCode that the type definition of exclusion is as follows:

```
interface ExclusionValidatorOptions extends WithOptionalMessage {
    /**
     * The list of values that cannot be used.
     */
    in?: Array<unknown>;
}
```

Should we update the documentation to be consistent with the implementation or vice versa?

### Are you interested in working on this?

- [X] I'm interested in working on this

**Comment by jtoar** [comment]
@taivo it looks like the types may not be flexible enough; the implementation supports the documented syntax: 

https://github.com/redwoodjs/redwood/blob/6c4a950876f44154c644849e030c95d1e661e84a/packages/api/src/validations/validations.ts#L303-L318

I'll get confirmation from @cannikin, but I'd say the types need updated if they're suggesting that `exclusion: [...]` is an error.

**Comment by cannikin** [comment]
I'm not the Type Guy so I couldn't really say what's wrong here! As @jtoar says it seems like `Array | ExclusionValidatorOptions` should be correct?

**Comment by jtoar** [comment]
@taivo looks like we need a fix here indeed—are you still up for giving this one a shot?

**Comment by taivo** [comment]
Right now this is what I'm seeing in validations.d.ts on my machine (Redwood 4.0.0 but it was like this for 3.8 as well). Sure, I will take a look. Perhaps I can learn something about type export here as well.

<img width="465" alt="Screenshot 2023-02-03 at 2 54 06 PM" src="https://user-images.githubusercontent.com/1006076/216726460-f961a03d-508f-46b9-99e0-cf67e7e5e492.png">

**Comment by taivo** [comment]
Ok. Fixed via #7565 . It looks like this type error escaped screening because the tests are written in javascript and not typescript. I temporarily changed the validation test file to `.ts` on my laptop and was able to see the type errors in VSCode.
I went ahead and fixed 2 other low hanging fruits in addition to the `inclusion` and `exclusion` types. Still, there are a few more type errors that I'm leaving for a future PR to keep this one review-friendly.

FYI @jtoar @cannikin
